pkg/pillar/cmd/downloader/download.go

```diff
@@ -16,7 +16,7 @@ import (
 func download(ctx *downloaderContext, trType zedUpload.SyncTransportType,
 	status Status, syncOp zedUpload.SyncOpType, downloadURL string,
```

Comment:
How about adding a comment specifying the return values?

Reply:
Sure

Reply:
Done